src/main/kotlin/com/ampnet/payoutservice/repository/JooqCreatePayoutTaskRepository.kt

```diff
@@ -35,6 +36,30 @@ class JooqCreatePayoutTaskRepository(private val dslContext: DSLContext, private
             ?.toModel()
     }
 
+    override fun getAllByChainIdIssuerOwnerAndStatuses(
+        chainId: ChainId,
+        issuer: ContractAddress?,
+        owner: WalletAddress?,
+        statuses: Set<TaskStatus>
+    ): List<CreatePayoutTask> {
+        logger.info {
```

Comment:
```suggestion
        logger.debug {
```
I would rather have get query as debug log.